packages/notifier/src/notifier.js

```diff
@@ -151,9 +152,113 @@ export const makeNotifierKit = (...initialStateArr) => {
   return harden({ notifier, updater });
 };
 
+/**
+ * @template T
+ * @param {ERef<Subscriber<T>>} subscriberP
+ * @returns {Promise<Notifier<T>>}
+ */
+export const makeNotifierFromSubscriber = async subscriberP => {
```

Comment:
This isn't an async function:
```suggestion
export const makeNotifierFromSubscriber = subscriberP => {
```

I don't think it needs to return `Promise` either. Can't it return the `Notifier` as `makeNotifierFromAsyncIterable` does?